+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Wed Aug 30 02:55:15 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL osinfo-db-tools
+ pkgs=osinfo-db-tools
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs osinfo-db-tools --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpv9pq7e5g
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpv9pq7e5g
++ pwd
+ pwd=/root/mugen-riscv/testcases/cli-test/libosinfo/oe_test_osinfo-db-import
+ LOG_INFO 'Finish preparing the test environment.'
+ message='Finish preparing the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish preparing the test environment.'
Wed Aug 30 02:55:18 2023 - INFO  - Finish preparing the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Wed Aug 30 02:55:18 2023 - INFO  - Start to run test.
+ wget https://releases.pagure.org/libosinfo/osinfo-db-20230308.tar.xz
oe_test_osinfo-db-import.sh: line 31: wget: command not found
+ osinfo-db-import --local osinfo-db-20230308.tar.xz
osinfo-db-import: cannot open archive /root/mugen-riscv/testcases/cli-test/libosinfo/oe_test_osinfo-db-import/osinfo-db-20230308.tar.xz: Failed to open '/root/mugen-riscv/testcases/cli-test/libosinfo/oe_test_osinfo-db-import/osinfo-db-20230308.tar.xz'
+ CHECK_RESULT 1 0 0 'Failed to import data'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed to import data'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed to import data'
+ LOG_ERROR 'Failed to import data'
+ message='Failed to import data'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed to import data'
Wed Aug 30 02:55:18 2023 - ERROR - Failed to import data
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_osinfo-db-import.sh line 33'
+ message='oe_test_osinfo-db-import.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_osinfo-db-import.sh line 33'
Wed Aug 30 02:55:18 2023 - ERROR - oe_test_osinfo-db-import.sh line 33
+ return 0
+ mkdir testdir
+ cd testdir
+ osinfo-db-export
osinfo-db-export: cannot get file info /etc/osinfo: Error when getting information for file “/etc/osinfo”: No such file or directory
+ CHECK_RESULT 1 0 0 'Failed to export data'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed to export data'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed to export data'
+ LOG_ERROR 'Failed to export data'
+ message='Failed to export data'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed to export data'
Wed Aug 30 02:55:18 2023 - ERROR - Failed to export data
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_osinfo-db-import.sh line 37'
+ message='oe_test_osinfo-db-import.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_osinfo-db-import.sh line 37'
Wed Aug 30 02:55:18 2023 - ERROR - oe_test_osinfo-db-import.sh line 37
+ return 0
+ test -e osinfo-db-20230331.tar.xz
+ CHECK_RESULT 1 0 0 'Export file does not exist'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Export file does not exist'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Export file does not exist'
+ LOG_ERROR 'Export file does not exist'
+ message='Export file does not exist'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Export file does not exist'
Wed Aug 30 02:55:18 2023 - ERROR - Export file does not exist
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_osinfo-db-import.sh line 39'
+ message='oe_test_osinfo-db-import.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_osinfo-db-import.sh line 39'
Wed Aug 30 02:55:18 2023 - ERROR - oe_test_osinfo-db-import.sh line 39
+ return 0
+ LOG_INFO 'End of the test.'
+ message='End of the test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of the test.'
Wed Aug 30 02:55:18 2023 - INFO  - End of the test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 3
+ test 3 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Wed Aug 30 02:55:18 2023 - ERROR - failed to execute the case.
+ exit 3
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Wed Aug 30 02:55:18 2023 - INFO  - Start to restore the test environment.
+ cd /root/mugen-riscv/testcases/cli-test/libosinfo/oe_test_osinfo-db-import
+ rm -rf testdir osinfo-db-20230308.tar.xz
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpv9pq7e5g ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpv9pq7e5g
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish restoring the test environment.'
+ message='Finish restoring the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish restoring the test environment.'
Wed Aug 30 02:55:18 2023 - INFO  - Finish restoring the test environment.
